build/crds-gen.sh

```diff
@@ -43,9 +45,17 @@ fi
 
 echo "Generating CRDs for API's..."
 
+allsed() {
```

Comment:
@timothymathison you can probably invoke gsed on MacOS OS type and fails in not found, as if `gsed` not installed and `sed` installed script will work incorrectly.

Reply:
@dmvolod, let me know what you think of it now.

Reply:
We should validate `gsed` presence before operator-sdk generate to avoid unpredictable changes

Reply:
@timothymathison looks good, thanks